tensorrt_llm/executor/worker.py

```diff
@@ -872,9 +876,16 @@ def _get_params_for_first_rsp(
     return None, None
 
 
-def _get_logprobs(worker, response: tllm.Response) -> Optional[LogProbsResult]:
+def _get_logprobs(worker,
+                  response: tllm.Response,
+                  is_pytorch_backend=False) -> Optional[LogProbsResult]:
     """Compute logprob and prompt logprob and clear out logits if applicable.
     """
+    if is_pytorch_backend:
```

Comment:
When PyT is ready, will it share this method and the `LogProbsResult`?

Reply:
IMO PyT path should use `LogProbsResult` even now to be consistent w/ TRT backend. I've asked Yuan for a follow-up PR for PyT.